cloudbuild.yaml

```diff
@@ -256,12 +257,16 @@ steps:
         python3 test_frontend.py "127.0.0.1:8081"
         echo "pass" > /workspace/rag_frontend_result.txt
 
-        # Upload locally stored netflix dataset to GCS bucket mounted as /data
-        gsutil cp ./netflix_titles.csv gs://gke-aieco-rag-$SHORT_SHA-$_BUILD_ID/netflix-shows/netflix_titles.csv
-        ray job submit --working-dir . --address=http://127.0.0.1:8265 -- python ray_job.py
-
-        python3 test_rag.py "http://127.0.0.1:8081/prompt"
```

Comment:
re-add this line and the one below after the notebook execution

Reply:
Added them back